xero_files.yaml

```diff
@@ -112,7 +113,7 @@ paths:
       summary: Retrieves a file by a unique file ID
       operationId: getFile
       parameters: 
-        - name: FileId
+        - name: FileID
```

Comment:
@SidneyAllen  - I didn't encounter this in Ruby, but the ID's non capitalized don't match the URL for the replacement.

`FileId` and `FileID` mismatch

```javascript
const localVarPath = this.basePath + '/Files/{FileID}/Associations'.replace('{' + 'FileId' + '}', encodeURIComponent(String(fileId)));
```

Does this look good? Want to make sure this does not effect others.

https://github.com/XeroAPI/Xero-OpenAPI/pull/362/files#diff-c9626d349308f5681c6a4c67412973121e9a94a3e528cd39415127cd41e70c65R273